modules/pulsar/pulsar_test.go

```diff
@@ -80,7 +88,7 @@ func TestPulsar(t *testing.T) {
 			name: "with log consumers",
 			opts: []testcontainers.ContainerCustomizer{
 				// withLogconsumers {
-				testcontainers.WithLogConsumers(&testcontainers.StdoutLogConsumer{}),
+				testcontainers.WithLogConsumers(&noopLogConsumer{}),
```

Comment:
We are changing here the default log consumer to avoid verbosity in the test output, right? Do you see any inconvenient with that?

Reply:
Yes, it made it difficult to debug issues with all the log output.